src/components/fieldset/index.md.njk

```diff
@@ -29,6 +29,14 @@ The first element inside a fieldset must be a `legend` which describes the group
 
 On [question pages](../../patterns/question-pages) containing a group of inputs, including the question as the legend helps users of screenreaders to understand that the inputs are all related to that&nbsp;question.
 
+### Legend as a page heading
```

Comment:
Should this go after "Include any general help text…"?

Reply:
I don't know if we have any prior art for heading up a section like this, but would it make sense to use a heading like 'When the legend is also the heading for the page'? /cc @amyhupe